### PR TITLE
Decrease MaxProposals for advisory committee

### PR DESCRIPTION
**Pull request by Chralt98**
Fixes partially this https://github.com/zeitgeistpm/zeitgeist/issues/822

### CODE REVIEW

**Review by maltekliemann — APPROVED** [review]
:thumbsup:

I'm just assuming that we won't have 256 or more proposals in the Advisory Committee when this update goes through.

**Comment by maltekliemann** [comment]
Nice catch, by the way!

File: runtime/zeitgeist/src/parameters.rs [hunk 1]
```diff
@@ -58,7 +58,8 @@ parameter_types! {
     // Collective
     // Note: MaxMembers does not influence the pallet logic, but the worst-case weight estimation.
     pub const AdvisoryCommitteeMaxMembers: u32 = 100;
-    pub const AdvisoryCommitteeMaxProposals: u32 = 300;
+    // The maximum of proposals is currently u8::MAX otherwise the pallet_collective benchmark fails
+    pub const AdvisoryCommitteeMaxProposals: u32 = 255;
```
[the remark this answers is not in the document]
Comment (Chralt98) [reply runtime/zeitgeist/src/parameters.rs:62]: Good assumption. `ProposalIndex` comes from substrate and is `u32`. So it's actually Substrate's fault, that the benchmark fails.

[the remark this answers is not in the document]
Comment (Chralt98) [reply runtime/zeitgeist/src/parameters.rs:62]: https://github.com/paritytech/substrate/blob/3cb5a4069974cdfae3f6aad18626e1dde19ce3fb/frame/collective/src/lib.rs#L75

[the remark this answers is not in the document]
Comment (Chralt98) [reply runtime/zeitgeist/src/parameters.rs:62]: I could probably do this `pub const AdvisoryCommitteeMaxProposals: u32 = 255u8 as u32;`, but it seems confusing.

**Pull request merged**